src/main/java/com/artipie/rpm/pkg/HeaderTags.java

```diff
@@ -260,6 +260,34 @@ public String sourceRmp() {
             .map(Flags::find).collect(Collectors.toList());
     }
 
+    /**
+     * Get the conflicts name header.
+     * @return Value of header tag CONFLICTNAME.
+     */
+    public List<String> conflicts() {
+        return this.meta.header(Header.HeaderTag.CONFLICTNAME).asStrings();
+    }
+
+    /**
+     * Get the conflicts versions header.
+     * @return Value of header tag CONFLICTVERSION.
+     */
+    public List<HeaderTags.Version> conflictsVer() {
+        return this.meta.header(Header.HeaderTag.CONFLICTVERSION).asStrings()
+            .stream().map(HeaderTags.Version::new).collect(Collectors.toList());
+    }
+
+    /**
+     * Get the conflicts flags header.
+     * @return Value of header tag CONFLICTFLAGS.
+     */
+    public List<Optional<String>> conflictsFlags() {
+        // @checkstyle MagicNumberCheck (2 lines)
+        return Arrays.stream(this.meta.header(Header.HeaderTag.CONFLICTFLAGS).asInts()).boxed()
+            .map(flag -> flag &= 0xf)
```

Comment:
First: `flag &= 0xf` doesn't make sense here, since `flag` is a parameter value and is not used anymore, it could be replaced with `flag & 0xf`.

Second: why apply this operation on boxed `Integer` instead of `int`? Mapping `IntStream` would be more effective here, since current map implementation will unbox and box it back, compiler will produce:
```java
map(new Function<Integer, Integer>() {
  Integer apply(Integer val) {
    // 1. unbox val to apply `&`
    // 2. box it back to return `Integer`
    return Integer.valueOf(val.intValue() & 0xf);
  }
}
```